SupportBank.Console/Program.cs

```diff
@@ -1,11 +1,244 @@
-﻿namespace SupportBank.Console
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using CsvHelper;
+using Newtonsoft.Json.Linq;
+using NLog;
+using NLog.Config;
+using NLog.Targets;
+
+namespace SupportBank.Console
 {
-  class Program
-  {
-    static void Main()
+    class Program
     {
-      // Please try to solve the problem yourself before looking at the model answer.
-      // Your trainer will work through the interesting bits of that solution in due course.
+        private static readonly ILogger logger = LogManager.GetCurrentClassLogger();
+        private static Dictionary<string, Person> people = new Dictionary<string, Person>();
+        private static List<Transaction> transactions = new List<Transaction>();
+
+        // Prints everyone's balance
+        private static void ListAll()
+        {
+            foreach (var person in people)
+                System.Console.WriteLine(person.Key + ": " + Math.Round(person.Value.Balance, 2));
+        }
+
+        // Prints name's transactions 
+        private static void ListAccount(string name)
+        {
+            if (!people.ContainsKey(name))
+            {
+                System.Console.WriteLine(name + " has no transactions");
+                return;
+            }
+
+            System.Console.WriteLine(name + ": " + people[name]);
+            foreach (var transaction in transactions.Where(x => x.From == name || x.To == name).ToList())
+            {
+                System.Console.WriteLine(transaction.ToString());
+            }
+        }
+
+        private static List<Transaction> ParseCSV(string filePath)
```

Comment:
It would be cleaner to have a `IInputParser` interface with a `ParseInput(string filePath)` method, and have the 3 Parser classes implement that